c/httpserver.c

```diff
@@ -2840,6 +2840,7 @@ static int getUserSessionValidity(char *username, const HttpServerConfig *config
       int currentValiditySec = 0;
       for (int i = 0; i < groupCount; i++) {
         retVal = getGroupSessionValidity(POINTER_FROM_INT(groups[i]), config, &currentValiditySec, returnCode, reasonCode);
+
```

Comment:
Do we need this change?

Reply:
Irek,

This is not from me, I do not why it's there.  I only add code to datasetjson.c.  Would it be merged when I did the PR?

Thanks.